packages/document/main-doc/docs/en/guides/concept/builder.mdx

```diff
@@ -4,31 +4,31 @@ sidebar_position: 2
 
 # Builder
 
-**Modern.js uses [Modern.js Builder](https://modernjs.dev/builder) to build your Web App.**
+**Modern.js's build capability is provided by [Modern.js Builder](https://modernjs.dev/builder).**
 
-Modern.js Builder is one of the core components of Modern.js. It is A build engine for web development. and can be used independently of Modern.js. Modern.js Builder supports multiple bundlers such as webpack and Rspack, and it uses webpack by default.
+Modern.js Builder is one of the core components of the Modern.js system. It is a build engine for web development scenarios that can be used independently of Modern.js. Modern.js Builder supports multiple packaging tools such as webpack and Rspack, and by default uses the most mature webpack for packaging.
 
 ## Build Architecture
 
-From the perspective of building, Modern.js is divided into three layers, from top to bottom:
+From a building perspective, Modern.js is divided into three layers, from top to bottom:
 
-- Upper-layer framework: Modern.js.
-- Universal build engine: Modern.js Builder.
-- Low-level bundlers: webpack and rspack.
+- Upper-level development framework: Modern.js.
+- Common build engine: Modern.js Builder.
+- Lower-level packaging tool: webpack and Rspack.
 
 <img src="https://lf3-static.bytednsdoc.com/obj/eden-cn/zq-uylkvT/ljhwZthlaukjlkulzlp/builder-layers-1117.png" style={{ width: '100%', maxWidth: '540px' }} />
 
 ## Builder Documentation
 
-Since Modern.js Builder is a module that can be used independently, we provide a separate document for it, the site address is: [modernjs.dev/builder](https://modernjs.dev/builder).
+Since Modern.js Builder is a module that can be used independently, we have provided separate documentation for it, which can be found at [modernjs.dev/builder](https://modernjs.dev/builder).
```

Comment:
```suggestion
Since Modern.js Builder is a module that can be used independently, we have provided separate documentation for it, which can be found at [modernjs.dev/builder](https://modernjs.dev/builder/en).
```